localstack/services/opensearch/cluster.py

```diff
@@ -118,6 +120,67 @@ def build_cluster_run_command(cluster_bin: str, settings: CommandSettings) -> Li
     return [cluster_bin] + cmd_settings
 
 
+def register_cluster(host, path, forward_url, custom_endpoint) -> List[Rule]:
```

Comment:
It would be great to have type hints here and a docstring which describes the expectations on the different parameters and its relation to `build_cluster_endpoint` (see comment in the tests).

Reply:
:+1: